enhanced_search/package.json

```diff
@@ -26,7 +27,7 @@
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "run-script-os",
+    "build": "run-script-os; cp node_modules/mark.js/dist/mark.js build/; cp src/content_scripts/* build/;",
```

Comment:
@rdestefa Don't remember if you're on windows or mac. Not sure if this line will work for you if you're on windows, so might need to move it to the build:windows and tweak it slightly